vagrant-tools/run-tests.sh

```diff
@@ -0,0 +1,99 @@
+. ~/.bash_profile
+cd /vagrant/borg/borg
+. ../borg-env/bin/activate
+if which pyenv > /dev/null; then
+  # for testing, use the earliest point releases of the supported python versions:
+  pyenv global 3.4.0 3.5.0
+fi
+# otherwise: just use the system python
+
+testingPartitionSize=128M
```

Comment:
I don't know, and I would like to support future tests. The FS minimum size is usually small, around 10M, so I don't think that's a big problem. Maybe we should drop this to 16M? I'll check that